src/lib/message/L1ToL2Message.ts

```diff
@@ -127,16 +127,13 @@ export abstract class L1ToL2Message {
       return ethers.utils.stripZeros(value.toHexString())
     }
 
-    const addressAlias = new Address(fromAddress)
-
-    const from = addressAlias.applyAlias()
```

Comment:
sure we don't need to apply an alias here anymore?
If the user is supplying the `from` address as their L1 address it still needs to be aliased.

The difference is that now the alias is applied int he L1 inbox instead of by arb-os.
We should clarify what's expected in the `fromAddress` field. If its the value from the Inbox delivered event, it doesn't need aliasing. If its the plain user address, it needs alaising

Reply:
ah, `calculateSubmitRetryableId` is public static
I think we should just add some comment regarding the input expected.